logstash-devutils.gemspec

```
@@ -35,12 +37,14 @@ Gem::Specification.new do |spec|
   spec.add_runtime_dependency "rake" # MIT License
   spec.add_runtime_dependency "gem_publisher" # MIT License
   spec.add_runtime_dependency "minitar" # GPL2|Ruby License
-  spec.add_runtime_dependency "logstash-core-plugin-api", "<= 2.99", ">= 2.0"
 
-  # Should be removed as soon as the plugins are using insist by their
-  # own, and not relying on being required by the spec helper.
-  # (some plugins does it, some use insist throw spec_helper)
-  spec.add_runtime_dependency "insist", "1.0.0" # (Apache 2.0 license)
+  # NOTE: as long as logstash-core-plugin-api forces a fixed logstash-core version dependency
+  # we should not add it here - we indirectly enforce LS >= 6 with the Ruby >= 2.3 requirement.
+  #spec.add_runtime_dependency "logstash-core", ">= 6", "< 9.0"
```

```suggestion
  spec.add_runtime_dependency "logstash-core", ">= 6.3"
```

Since we otherwise note in this PR that this works back to Logstash 6.3, I'm a fan of making that requirement explicit. I don't think we need to pin to <9 though, since we haven't even released 8 and we don't know what the future holds in terms of breaking changes.

---

:+1: would love to do that ...
a bit problematic to accomplish as long as plugins depend on logstash-core-plugin-api
... the last [published version has a hard dependency on logstash-core 5.6.4](https://rubygems.org/gems/logstash-core-plugin-api/versions/2.1.28-java)

guess if we force the plugin to always use a LS install (thus a path: source) -> should work

---

in which case having logstash-core-plugin-api as a dependency on devutils makes little sense.
... its mostly useful for plugins and having only logstash-core dependency specified should do.